Recruit-Training/Basic-Recruit-Training/pointers/main.cpp

```diff
@@ -11,6 +11,7 @@ using namespace std;
 
 void foo(int *a, int b);
 void bar(int *a, int b);
+void Myswap(int *a,int*b);
```

Comment:
make the spacing with function Myswap similar to the functions above it.In this case it would be like 
"void Myswap(int *a, int *b)